hack/local-up-cluster.sh

```diff
@@ -400,6 +400,44 @@ cleanup()
   exit 0
 }
 
+# Check if all processes are still running. Prints a warning once each time
+# a process dies unexpectedly.
+healthcheck()
```

Comment:
minor nit: for consistency with the rest of the file, use `function healthcheck {` instead.

Reply:
Fixed.